File: frontend/src/types/weather.ts
export interface DayInfo {
	_id: string;
	dayNumber: number;
	weekday: string;
	month: string;
	year: number;
	hour: string;
}

export interface WeatherLogFull {
	_id: string;
	time: string;
	temperature: number;
	humidity: number;
	wind_speed: number;
	rain_probability: number;
	weather_code: string;
	sky_condition: string;

	dayNumber: number;
	weekday: string;
	month: string;
	year: number;
	hour: string;
}

File: queue-worker-go/core/domain/weather.go
package domain

type WeatherDocument struct {
	Time             string  `json:"time"`
	Type             string  `json:"type"`
	UserId 			 string  `json:"userId"`
	Temperature      float64 `json:"temperature"`
	TempMax          float64 `json:"temp_max"`
	TempMin          float64 `json:"temp_min"`
	Humidity         float64 `json:"humidity"`
	WindSpeed        float64 `json:"wind_speed"`
	RainProbability  float64 `json:"rain_probability"`
	WeatherCode      int     `json:"weather_code"`
	SkyCondition     string  `json:"sky_condition"`
}